_includes/scenarios/integrate/establish-a-relationship-to-another-identity.md

```diff
@@ -0,0 +1,164 @@
+
+{% include copy-notice description="Read the `id` of the Relationship from the `relationships.id` property and the `id` of the RelationshipChange from the `relationships.changes.id` property of the synchronization `result` for the next step." %}
+
+### Accept the Relationship Request
+
+If the Templator accepts the Relationship Request, the `status` of the data object of type [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) will change from `"Pending"` to `"Active"` and therefore an active Relationship between the Templator and the Requestor will be established. To do this, consult the [Accept RelationshipChange]({% link _docs_use-cases/use-case-transport-accept-relationship-change.md %}) use case description and specify the `id` of the Relationship and the `id` of the RelationshipChange.
+
+For rejecting the Relationship Request and therefore not establishing an active Relationship between the Templator and the Requestor, take a look at the documentation of the [Reject RelationshipChange]({% link _docs_use-cases/use-case-transport-reject-relationship-change.md %}) use case.
+{: .notice--info}
+
+Assuming the Requestor is a Connector, it must [synchronize the updates of the Backbone]({% link _docs_use-cases/use-case-transport-synchronize-updates-of-backbone.md %}) after the Templator has accepted the Relationship Request. The `result` of the response after synchronization shows in particular that the `status` of the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) has been changed from `"Pending"` to `"Active"` and that the `status` of the associated [RelationshipChange]({% link _docs_integrate/data-model-overview.md %}#relationshipchange) with `"Creation"` as `type` has been changed from `"Pending"` to `"Accepted"`. Now the Requestor is informed that the Templator has accepted the Relationship Request and therefore an active Relationship has been established between them.
+
+## What's next?
+
+After an active Relationship between the two Identities is established, they are able to share information with each other. For example, they can exchange Messages. How a Connector can send a Message to another Identity with which it has an active Relationship is described in the [Integration Example]({% link _docs_integrate/integration-example.md %}).
```

Comment:
Maybe, rather link to "Requests over Messages" here or another Scenario that requires an active Relationship, since in the Integration Example you start by establishing a Relationship again.